control/veda_deliver_cielo.py

```diff
@@ -1,186 +1,227 @@
-
-import os
-import sys
-import requests
-from requests.auth import HTTPBasicAuth
+"""
+Cielo24 Integration
+"""
 import ast
+import logging
+import sys
 import urllib
 
-"""
-Cielo24 API Job Start and Download
+import requests
+from django.core.urlresolvers import reverse
 
-Options (reflected in Course.models):
-transcription_fidelity =
-  Mechanical (75%),
-  Premium (95%)(3-72h),
-  Professional (99+%)(3-72h)
+from control.veda_val import VALAPICall
+from VEDA_OS01.models import (URL, TranscriptProcessMetadata,
+                              TranscriptProvider, TranscriptStatus, Video,
+                              VideoTranscriptStatus)
+from VEDA_OS01.utils import build_url
 
-priority =
-  standard (24h),
-  priority (48h)
+requests.packages.urllib3.disable_warnings()
```

Comment:
we're aiming to disable SSL `InsecurePlatformWarning` i.e.
 ```
InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL
appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve
this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
```

we should disable only that warning:
```
from requests.packages.urllib3.exceptions import InsecurePlatformWarning
requests.packages.urllib3.disable_warnings(InsecurePlatformWarning)
```